pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -21,10 +21,10 @@
 , "dhcp"
 , "dnschain"
 , "elasticdump"
-, "elm-doc-preview"
+// , "elm-doc-preview" - surpassed by elmPackages.elm-doc-preview https://github.com/NixOS/nixpkgs/pull/62899
 , "elm-live"
 , "elm-oracle"
-, "elm-test"
+// , "elm-test" - surpassed by elmPackages.elm-doc-preview https://github.com/NixOS/nixpkgs/pull/62899
```

Comment:
should be probably removed but I feel regenerating all node10 deps is out of scope of this PR.

Reply:
JSON doesn't support comments so you can just remove them and next time someone regenerates it will be gone.

Reply:
yes, I thought this will force people to remove it but at least it would be clear why it disappeared from generated files. Anyway, I can remove it :+1: